geoportailv3/static/js/wmsservice.js

```diff
@@ -181,10 +181,16 @@ app.WmsHelper.prototype.buildChildLayers_ = function(wms, layer, wmsVersion,
     layer['useTiles'] = useTiles;
   }
 
+  // casacading CRS
+  layer.CRS = layer.CRS || [];
+  if(parentLayer) {
+    parentLayer.CRS.forEach(crs => layer.CRS.indexOf(crs) < 0 && layer.CRS.push(crs));
```

Comment:
this code isn't transpiled, right? So you can't use arrow functions because of [IE11](https://caniuse.com/#feat=arrow-functions)